interp/memory.go

```diff
@@ -196,6 +196,8 @@ func (mv *memoryView) markExternal(llvmValue llvm.Value, mark uint8) {
 				element := llvm.ConstExtractValue(llvmValue, []uint32{uint32(i)})
 				mv.markExternal(element, mark)
 			}
+		case llvm.PointerTypeKind:
+			// Do nothing
```

Comment:
This is incorrect, although it may appear to work. I'll look into it and try to find a good solution.

Reply:
@kenbell : 

For my education, why did you add this extra code ?

Reply:
It's a workaround, here is a fix: https://github.com/tinygo-org/tinygo/pull/1891

Reply:
Thanks @aykevl - I'll take a look at #1891